x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvMaxTests.java

```diff
@@ -44,6 +44,7 @@ public static Iterable<Object[]> parameters() {
             (size, values) -> equalTo(NumericUtils.asLongUnsigned(values.reduce(BigInteger::max).get()))
         );
         dateTimes(cases, "mv_max", "MvMax", (size, values) -> equalTo(values.max().getAsLong()));
+        geoPoints(cases, "mv_max", "MvMax", (size, values) -> equalTo(values.max().getAsLong()));
```

Comment:
I don't think that makes a lot of sense, do we need to support it?

Reply:
Good question. It was simplest to code this, rather than explicitly reject these, but we can discuss alternatives. I think, however, that a deterministic ordering of all points is something that might be necessary anyway, and in that case ordering by the doc-value is as good a choice as any (order by lat first then lon second).